src/jquery.vide.js

```diff
@@ -313,6 +362,17 @@
       // Resize a video, when it's loaded
       .one('canplaythrough.' + PLUGIN_NAME, function() {
         vide.resize();
+
+        if (vide.settings.pauseWhenHidden) {
+          var visibilitySupport = detectVisibilitySupport();
+
+          if (visibilitySupport) {
+            hiddenPropertyName = visibilitySupport[0];
+            $(document).on(visibilitySupport[1], function() {
+              handleVisibilityChange(vide.$video[0]);
```

Comment:
The logic of the function is not complecated, seems like `handleVisibilityChange` is unnecessary.